ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__installer-rehearse-4.19.yaml

```diff
@@ -37,6 +37,24 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: installer-rehearse-manual-gcp-confidential
+  cron: '@yearly'
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_NODE_TYPE: c3-standard-4
+      CONFIDENTIAL_COMPUTE: IntelTrustedDomainExtensions
+      CONTROL_PLANE_CONFIDENTIAL_COMPUTE: AMDEncryptedVirtualization
```

Comment:
Suggest to use confidential computing type `AMDEncryptedVirtualizationNestedPaging` (which is the new one supported in 4.19) along with machine type `n2d-standard-4`.

Reply:
BTW, will you update step `ipi-install-install` to save the kubeconfig?

Reply:
>Suggest to use confidential computing type AMDEncryptedVirtualizationNestedPaging (which is the new one supported in 4.19) along with machine type n2d-standard-4.

OK, let me update.

Reply:
> kubeconfig is always copied to SHARED_DIR, we need to login the pod and get it. It cannot be printed in build log for security reason

+1, when step `cucushift-installer-wait` is running, we can copy `kubeconfig` file from that pod.